opm/parser/eclipse/EclipseState/Tables/SimpleTable.cpp

```diff
@@ -153,6 +153,16 @@ void SimpleTable::checkMonotonic(const std::string& columnName,
     }
 }
 
+void SimpleTable::checkRange(const std::string& columnName)
```

Comment:
If you indeed intend to throw I would prefer if the method were named e.g. `assertRange()` - it should be safe to call a method named `checkRange()` without catching fire.